src/main.js

```diff
@@ -5,15 +5,15 @@ import DataModel from './model/points-model';
 import SortingModel from './model/sorting-model';
 import {generateFilter} from './mock/filter';
 
-const HEADER = document.querySelector('.page-header');
-const MAIN = document.querySelector('.page-main');
-const TRIP_CONTROL_ELEMENTS = HEADER.querySelector('.trip-controls__filters');
-const TRIP_ELEMENTS = MAIN.querySelector('.trip-events');
+const header = document.querySelector('.page-header');
+const main = document.querySelector('.page-main');
+const tripControlElements = header.querySelector('.trip-controls__filters');
+const tripElements = main.querySelector('.trip-events');
```

Comment:
По хорошему все элементы которые выбираются через querySelector именовать с окончанием Element
```suggestion
const headerElement = document.querySelector('.page-header');
const mainElement = document.querySelector('.page-main');
const tripControlElements = header.querySelector('.trip-controls__filters');
const tripElements = main.querySelector('.trip-events');
```